express/blocks/schemas/schemas.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import {
+  getLocale,
+} from '../../scripts/scripts.js';
+
+function decorateSchemasBlocks($block) {
+  const rows = Array.from($block.children);
+  const locale = getLocale(window.location);
+  const urlPrefix = locale === 'us' ? '' : `/${locale}`;
+  const homePageLocaleUrl = ''.concat('https://www.adobe.com/', urlPrefix, '/');
+  let webApplicationUrl;
+  rows.forEach(($row) => {
+    const cells = Array.from($row.children);
+    const cellValue = cells[0].innerText;
+    if (cellValue === '') {
+      webApplicationUrl = document.querySelectorAll('p.button-container')[0].getElementsByTagName('a')[0].href;
+    } else {
+      webApplicationUrl = cellValue;
+    }
+  });
+  const webPageSchemaScript = document.createElement('script');
+  webPageSchemaScript.setAttribute('type', 'application/ld+json');
+  const webSchemaJson = {
+    '@context': 'https://schema.org',
+    '@type': 'WebPage',
+    url: window.location.href,
+    '@id': ''.concat(window.location.href, '/#webpage'),
```

Comment:
nit: I think interpolation would be nicer here (and all other places where we used ''.concat) -- `${window.location.href}/#webpage`;

Reply:
GH removed my backticks... '``'